packages/site/cms-site/src/blocks/DamFileDownloadLinkBlock.tsx

```diff
@@ -5,29 +5,23 @@ import { withPreview } from "../iframebridge/withPreview";
 import { PropsWithData } from "./PropsWithData";
 
 interface Props extends PropsWithData<DamFileDownloadLinkBlockData> {
-    children: React.ReactNode;
+    children: React.ReactElement;
     title?: string;
 }
 
 export const DamFileDownloadLinkBlock = withPreview(
     ({ data: { file, openFileType }, children, title }: Props) => {
-        if (file === undefined) {
-            return <>{children}</>;
+        if (!file) {
+            return children;
         }
 
-        if (openFileType === "Download") {
-            return (
-                <a href={file.fileUrl} title={title}>
-                    {children}
-                </a>
-            );
-        } else {
-            return (
-                <a href={file.fileUrl} target="_blank" rel="noreferrer" title={title}>
-                    {children}
-                </a>
-            );
-        }
+        const childProps = {
+            href: file.fileUrl,
+            title,
+            ...(openFileType !== "Download" && { target: "_blank" }),
```

Comment:
It's better to have a positive check:

```suggestion
            ...(openFileType === "NewTab" && { target: "_blank" }),
```